/*
 * Copyright (C) 2023 The Android Open Source Project
 *
 * Licensed under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 *
 *      http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 */
package com.android.wallpaper.picker.preview.ui.viewmodel

import com.android.wallpaper.model.wallpaper.DeviceDisplayType
import com.android.wallpaper.util.PreviewUtils

/** Defines configuration associated with a single workspace preview. */
data class WorkspacePreviewConfigViewModel(

    /** The preview utils for rendering the workspace preview, different for Home & Lock screens. */
    val previewUtils: PreviewUtils,

    /** The foldable display to be rendered, or null if the device is not a foldable. */
    val deviceDisplayType: DeviceDisplayType,
)
